LICENSE

```diff
@@ -1,31 +1,22 @@
-BSD License
+MIT License
 
 For React software
 
 Copyright (c) 2013-present, Facebook, Inc.
-All rights reserved.
```

Comment:
This line was removed from LICENSE file, but not from other source files... :blush:

Reply:
Thanks, fixed.